### PR TITLE
add supported OS to fix tests

### PR DESCRIPTION
**Pull request by SimonHoenscheid**
rebase #244  after merge

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -32,25 +32,43 @@
...
+      "operatingsystemrelease": [
+        "8",
+        "9"
       ]
     },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "7"
+        "8",
```
Comment (h-haaks) [inline metadata.json:56]: CentOS here accually refers to CentOS Stream, and CentOS Stream8 is EOL as well

**Comment by h-haaks** [comment]
If you only add new os releases in this PR and don't remove any This PR could be merged and #244 rebased to have green CI there as well before merge.

File: metadata.json (hunk 1 above)
Comment (SimonHoenscheid) [reply metadata.json]: I removed it

File: metadata.json [hunk 2]
```diff
@@ -32,25 +32,45 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7"
```
Comment (h-haaks) [inline metadata.json:35]: Redhat 7 is still removed here :)

**Review by h-haaks — APPROVED** [review]

**Comment by h-haaks** [comment]
Change first message before merge?

**Comment by h-haaks** [comment]
And maybe update->add in the title?

**Pull request merged**